internal/validations/clusterlogforwarder/outputs/azuremonitor_test.go

```diff
@@ -41,7 +41,7 @@ var _ = Describe("Validate AzureMonitor:", func() {
 			}
 			valid, st := VerifyAzureMonitorLog("azureMonitor", azureMonitor)
 			Expect(valid).To(BeFalse())
-			Expect(map[string]status.Condition{"azureMonitor": st}).To(HaveCondition("Ready", false, loggingv1.ReasonInvalid, "output \"azureMonitor\": LogType can only contain letters, numbers, and underscores \\(_\\), and may not exceed 100 characters."))
+			Expect(map[string]status.Condition{"azureMonitor": st}).To(HaveCondition("Ready", false, loggingv1.ReasonInvalid, "output \"azureMonitor\": LogType names must start with a letter/number, contain only letters/numbers/underscores \\(_\\), and be between 1-100 characters."))
```

Comment:
no need to change... but you could also use back ticks instead of quotes for the string and then there would be no need to escape the internal quotes